google-cloud-bigtable/pom.xml

```diff
@@ -39,9 +39,9 @@
 
     <!-- These are needed to compile the protobuf used by Changestream merging acceptance test. We have enforcer rule
      below to make sure that the versions here match the ones we pull in via the shared bom. -->
```

Comment:
Please update the comment as well :)

it looks like test-protobuf.version is not used, maybe inline its value in test-protoc.version and remove it

Reply:
done